src/internal/minikubetestenv/deploy.go

```diff
@@ -396,8 +396,16 @@ func waitForLoki(t testing.TB, lokiHost string, lokiPort int) {
 }
 
 func waitForPgbouncer(t testing.TB, ctx context.Context, kubeClient *kube.Clientset, namespace string) {
+	waitForLabeledPod(t, ctx, kubeClient, namespace, "app=pg-bouncer")
+}
+
+func waitForPostgres(t testing.TB, ctx context.Context, kubeClient *kube.Clientset, namespace string) {
```

Comment:
Yeah, I was aware of this potential but I think

- all of the other waitFor functions are affected by this as well, so it's not a new issue. I made a ticket to address this holistically in the future https://pachyderm.atlassian.net/browse/CORE-1536
- postgres goes into terminating very quickly, it just takes a while to spin down and back up. Since we check postgres last(after loki, pachd and pg-bouncer) the chance that postgres isn't at least going down is really small. Watching locally at least, we aren't even close to hitting this race condition.

I could attempt to find a way to get the `controller-revision-hash` and confirm it's good, but I think it's pragmatic to get this in and handle that in bulk with CORE-1536.